llarp/profiling.cpp

```diff
@@ -75,7 +75,8 @@ namespace llarp
   {
     if (connectTimeoutCount > chances)
```

Comment:
probably a good time to comment this about what it's supposed to be doing / why.

Reply:
excluded this hunk it's not needed.